solc/CommandLineInterface.cpp

```diff
@@ -414,25 +416,28 @@ void CommandLineInterface::readInputFilesAndConfigureRemappings()
 			else
 			{
 				auto infile = boost::filesystem::path(path);
-				if (!boost::filesystem::exists(infile))
+				if (!boost::filesystem::exists(infile) || !boost::filesystem::is_regular_file(infile))
 				{
-					cerr << "Skipping non-existent input file \"" << infile << "\"" << endl;
-					continue;
-				}
+					//missing or irregular file, no direction to ignore missing files from command line
+					if(! m_args.count(g_argIgnoreMissingFiles)){
```

Comment:
Please correct whitespace:

 - space after keywords
 - no space between unary operator and operand
 - `{` on a line of its own

Reply:
Fixed.